modules/snooker/src/test/ts/browser/TableAdjustmentsTest.ts

```diff
@@ -102,167 +144,243 @@ UnitTest.test('TableAdjustmentsTest', () => {
+      it('preserveTable cols (3)', testAdjustWidth(450, [ 112.5, 112.5, 112.5, 112.5 ], pixelTableWithColGroup(), 50, 3, preserveTable, true));
+      it('preserveTable overflow (3)', testAdjustWidth(450, [ 132.5, 101.5, 101.5, 101.5 ], pixelTableWithOverflow(), 50, 3, preserveTable, false));
+    });
+
+    context('ltr large step (px)', () => {
+      it('preserveTable cells (0)', testAdjustWidth(400, [ 183.5, 10, 96.75, 96.75 ], pixelTable(), 200, 0, preserveTable, false));
+      it('preserveTable cols (0)', testAdjustWidth(400, [ 190, 10, 100, 100 ], pixelTableWithColGroup(), 200, 0, preserveTable, true));
+      it('preserveTable cells (1)', testAdjustWidth(400, [ 96.75, 183.5, 10, 96.75 ], pixelTable(), 200, 1, preserveTable, false));
+      it('preserveTable cols (1)', testAdjustWidth(400, [ 100, 190, 10, 100 ], pixelTableWithColGroup(), 200, 1, preserveTable, true));
+      it('preserveTable cells (2)', testAdjustWidth(400, [ 96.75, 96.75, 183.5, 10 ], pixelTable(), 200, 2, preserveTable, false));
+      it('preserveTable cols (2)', testAdjustWidth(400, [ 100, 100, 190, 10 ], pixelTableWithColGroup(), 200, 2, preserveTable, true));
+      it('preserveTable cells (3)', testAdjustWidth( 600, [ 146.75, 146.75, 146.75, 146.75 ], pixelTable(), 200, 3, preserveTable, false));
+      it('preserveTable cols (3)', testAdjustWidth( 600, [ 150, 150, 150, 150 ], pixelTableWithColGroup(), 200, 3, preserveTable, true));
+    });
+
+    context('rtl step (px)', () => {
+      it('preserveTable cells (0)', testAdjustWidth(400, [ 46.75, 146.75, 96.75, 96.75 ], pixelTable(), -50, 0, preserveTable, false));
+      it('preserveTable cols (0)', testAdjustWidth(400, [ 50, 150, 100, 100 ], pixelTableWithColGroup(), -50, 0, preserveTable, true));
+      // TODO: TINY-7942: This needs design input as it should be blocked since it can't be resized smaller
+      it.skip('preserveTable overflow (0)', testAdjustWidth(400, [ 120, 89, 89, 89 ], pixelTableWithOverflow(), -50, 0, preserveTable, false));
+      it('preserveTable cells (1)', testAdjustWidth(400, [ 96.75, 46.75, 146.75, 96.75 ], pixelTable(), -50, 1, preserveTable, false));
+      it('preserveTable cols (1)', testAdjustWidth(400, [ 100, 50, 150, 100 ], pixelTableWithColGroup(), -50, 1, preserveTable, true));
+      it('preserveTable overflow (1)', testAdjustWidth(400, [ 120, 39, 139, 89 ], pixelTableWithOverflow(), -50, 1, preserveTable, false));
+      it('preserveTable cells (2)', testAdjustWidth(400, [ 96.75, 96.75, 46.75, 146.75 ], pixelTable(), -50, 2, preserveTable, false));
+      it('preserveTable cols (2)', testAdjustWidth(400, [ 100, 100, 50, 150 ], pixelTableWithColGroup(), -50, 2, preserveTable, true));
+      it('preserveTable overflow (2)', testAdjustWidth(400, [ 120, 89, 39, 139 ], pixelTableWithOverflow(), -50, 2, preserveTable, false));
+      it('preserveTable cells (3)', testAdjustWidth( 350, [ 84.25, 84.25, 84.25, 84.25 ], pixelTable(), -50, 3, preserveTable, false));
```

Comment:
In a few of these, it looks like there is an extra space before the first parameter

Reply:
I could only see the one which I left a suggestion for. Can you see any others?

Reply:
```suggestion
      it('preserveTable cells (3)', testAdjustWidth(350, [ 84.25, 84.25, 84.25, 84.25 ], pixelTable(), -50, 3, preserveTable, false));
```
Ahh, I see what you mean. This was actually like this as well before my changes here, but will get them fixed.